src/pages/WorkloadDetails/WorkloadInfo.tsx

```diff
@@ -114,7 +114,7 @@ class WorkloadInfo extends React.Component<WorkloadInfoProps, WorkloadInfoState>
                       {pods.length > 0 && <WorkloadPods pods={pods} validations={this.props.validations!['pod']} />}
                     </TabPane>
                     <TabPane eventKey={'services'}>
-                      {pods.length > 0 && <WorkloadServices services={services} namespace={this.props.namespace} />}
+                      {services.length > 0 && <WorkloadServices services={services} namespace={this.props.namespace} />}
```

Comment:
This is unrelated to this PR, did that other commit got mixed?

Reply:
@josejulio This came from your original legend PR.

Reply:
oh right, i think is related to this commit  https://github.com/kiali/kiali-ui/commit/db3c7b3e3f646c2b98ea21beb6e3d4f04e1bb7ee